src/Bicep.Cli/Services/CompilationService.cs

```diff
@@ -104,8 +107,23 @@ public ParamsSemanticModel CompileParams(string inputPath, bool skipRestore)
             {
                 throw new Exception($"Unable to read file {inputPath}");
             }
+            
+            var paramsFile = SourceFileFactory.CreateBicepParamFile(inputUri, fileText);
```

Comment:
I feel like I need to perform these steps to get a bicep file Uri (both in CLI and LS), should I move it to path helper or is duplication fine?